lib/jsx/getLayerPixmap.jsx

```diff
@@ -47,8 +47,7 @@
 //         (Default: false)
 //   - compId: number, layer comp ID (optionally and exclusive of compIndex)
 //   - compIndex: number, layer comp index (optionally and exclusive of compId) 
-
-var MAX_DIMENSION = 10000;
+//   - maxDimension: number, maximal dimension of pixmap in pixels
```

Comment:
You have this parameter in the "optional params" section, but it isn't actually optional (i.e. if someone uses this file separate from the `generator.getLayerPixmap` method, it won't work unless they supply a `params.maxDimension`.

I do agree it should be optional. The correct fix is probably to rename `MAX_DIMENSION` to `DEFAULT_MAX_DIMENSION` and then change line 133 and 134 to something like `params.maxDimension || DEFAULT_MAX_DIMENSION`